seed_stage_based_messaging/settings.py

```diff
@@ -144,6 +147,11 @@
     "dsn": STAGE_BASED_MESSAGING_SENTRY_DSN
 }
 
+CACHES = {
+    "default": env.cache(default="locmemcache://"),
+    "locmem": {"BACKEND": "django.core.cache.backends.locmem.LocMemCache"},
```

Comment:
We have ~14000 message in production, so I don't think the default `MAX_ENTRIES` of 300 will work for us here, I think around 30000 is probably a better value. Also, the default `TIMEOUT` of 5 minutes is probably too short, we could do a 60 minute timeout here without a problem.